### ISSUE TITLE
Typos & wording

### ISSUE THREAD

**Issue by rng-daemon** [compgenomr/book#19]
* 5.1 Enumeration  sometimes has periods at the end, sometimes not: 
1. Define a prediction function or method f(X)
2. Devise a function (called loss or cost function) to optimize the difference between your predictions and observed values, such as ∑(Y−f(X))2
3. Apply mathematical optimization methods to find best parameter values for f(X) in relation to the cost/loss function.

* 5.1.1 "However, while doing so[, the] field of statistics developed..."

* 5.2 Enumeration  sometimes has periods at the end, sometimes not. 
* "algorithm becomes relevant.[ ]“Training” generally"

* 5.4.2: `library(caret)` loaded to late/inconsistently
 
* 5.4.3: "Removing genes or samples have both downsides." Please ask English native speaker. Recommendation: 
"Both, removing genes and samples have downsides"

* 5.5.1: "For starters, we will split **the** 30% of the data as test." Which _the_?

* 5.7: "Accuracy is the first metric to look at. This metric is **is** simply..." Double _is_. 
* `# gte k-NN prediction on the training data itself, with k=5`. *gte*?

* 5.12 "Another variable we can tune is the minimum node size of terminal nodes in the trees (min.node.size). This controls the depth of the trees grown. Setting this to **larger** numbers might cost a small loss in accuracy but the algorithm will run faster." Shouldn't it mean smaller numbers?

**Comment by al2na** [comment]
Please provide your full name so I can acknowledge you. Thanks for the suggestions!!

**Comment by al2na** [comment]
changed what I can, some suggestions didn't make sense.
